grpc/src/proofs.rs

```diff
@@ -15,20 +15,30 @@ pub struct AggProofRequest {
 /// Response from an aggregation proof request
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct AggProofResponse {
-    #[prost(bool, tag="1")]
-    pub success: bool,
```

Comment:
We have discussed previously that success could be error code, for easier parsing of the error type on aggkit prover.